reference-data-proxy/src/database/mongo-client.ts

```diff
@@ -1,4 +1,4 @@
-const { MongoClient } = require('mongodb');
+import { MongoClient } from 'mongodb';
```

Comment:
I'm not sure if this works. locally it's throwing an error when we try to connect to the mongo client - on line 9